sections/02_intro.md

```diff
@@ -42,7 +43,7 @@ these steps. When sufficient data are available, these methods construct
 features tuned to a specific problem and combine those features into a
 predictor. Recently, hardware improvements and very
 large training datasets have allowed these deep learning techniques to surpass
-other machine learning algorithms for many problems.
+other machine learning algorithms for many important problems [@tag:Krizhevsky2013_nips_cnn @arxiv:1412.5567]. 
```

Comment:
new syntax for multiple citations (place semicolon in between)

Reply:
fixed